ui/mods/msu/css/settings_screen.css

```diff
@@ -241,52 +245,76 @@
 	width: 60.0rem;
 	height: 30.0rem;
 
-	background-image: url("coui://gfx/ui/skin/popup_background_600x300.png");
+	background-image: url("coui://gfx/mods/skin/generic_background_large_border.png");
```

Comment:
mods/msu, and we should convert our other stuff to be the same

Reply:
fair